### PR TITLE
fix file URLs on windows

### PR DESCRIPTION
**Pull request by zkat**

### CODE REVIEW

File: normalize-git-url.js [hunk 1]
```diff
@@ -23,8 +23,10 @@ module.exports = function normalize (u) {
   if (altered) {
     if (u.match(/^git\+https?/) && parsed.pathname.match(/\/?:[^0-9]/)) {
       returnedUrl = u.replace(/^git\+(.*:[^:]+):(.*)/, '$1/$2')
+    } else if (u.match(/^git\+file/)) {
+      returnedUrl = u.replace(/^git\+/, '')
     } else {
-      returnedUrl = u.replace(/^(?:git\+)?ssh:\/\//, '')
+      returnedUrl = u.replace(/^(?:git\+)?ssh:[\/\\]{2}/, '')
```
Comment (iarna) [inline normalize-git-url.js]: I was going to ask under what circumstances an ssh url would end up as `ssh:\\` but I guess it doesn't really matter. =p

Comment (zkat) [inline normalize-git-url.js]: oh. That. May have just been me thrashing around when first trying to fix this. I realized afterwards that slashes aren't just normalized quite that aggressively just because you're on windows :joy: -- I'll just take this bit off tbh.

Comment (iarna) [inline normalize-git-url.js]: tbh, I'm a little itchy over this line, since it seems entirely unrelated to the other changes here? Maybe have this as a separate commit w/ its own test?

**Comment by iarna** [comment]
LGTM
